.github/workflows/auto-merge-bot.yml

```
@@ -0,0 +1,22 @@
+on: [pull_request]
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      # To use this repository's private action,
+      # you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Enviornment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: false
+      - run: cd .github/bot && npm install && npm run build
```

```suggestion
      - run: cd .github/bot && npm ci && npm run build
```
`ci` will install exactly what `package-lock.json` says rather than whatever npm happens to return for each package.

---

Is it possible to move this install/build step into the action itself, rather than outside the action?

---

I'll try to removing ncc

---

Check out the PR I made against your repository, I *think* I may have already removed it (effectively), so you can just remove the dependency.

---

I think it'd be best to just do it in that PR (speaking of which, not sure how to make changes to your branch)

---

My PR is against your fork, if you are generally OK with the changes, I recommend just merging it and then working on your fork like you have been (or undoing things you don't like from the PR).